Author: Kevin Chevreuil - Kaisen <kaisen@kaisenlinux.org>

--- partman-auto-160.orig/recipes/atomic
+++ partman-auto-160/recipes/atomic
@@ -5,24 +5,20 @@ partman-auto/text/atomic_scheme ::
 	$reusemethod{ }
 	method{ biosgrub } .
 
-512 512 768 ext2
+1024 1024 1024 btrfs
 	$defaultignore{ }
 	method{ format }
 	format{ }
+	label{ KaisenBoot }
 	use_filesystem{ }
-	filesystem{ ext2 }
+	filesystem{ btrfs }
 	mountpoint{ /boot } .
 
-900 10000 -1 $default_filesystem
+10000 30000 -1 btrfs
 	$lvmok{ }
 	method{ format }
 	format{ }
+	label{ KaisenLinux }
 	use_filesystem{ }
-	$default_filesystem{ }
+	filesystem{ btrfs }
 	mountpoint{ / } .
-
-100% 512 200% linux-swap
-	$lvmok{ }
-	$reusemethod{ }
-	method{ swap }
-	format{ } .
--- partman-auto-160.orig/recipes/home
+++ partman-auto-160/recipes/home
@@ -5,33 +5,30 @@ partman-auto/text/home_scheme ::
 	$reusemethod{ }
 	method{ biosgrub } .
 
-512 512 768 ext2
+1024 1024 1024 btrfs
 	$defaultignore{ }
 	method{ format }
 	format{ }
+	label{ KaisenBoot }
 	use_filesystem{ }
-	filesystem{ ext2 }
+	filesystem{ btrfs }
 	mountpoint{ /boot } .
 
-1500 6000 30000 $default_filesystem
+10000 20000 40000 btrfs
 	$lvmok{ }
 	method{ format }
 	format{ }
+	label{ KaisenLinux }
 	use_filesystem{ }
-	$default_filesystem{ }
+	filesystem{ btrfs }
 	mountpoint{ / } .
 
-100% 512 200% linux-swap
-	$lvmok{ }
-	$reusemethod{ }
-	method{ swap }
-	format{ } .
-
-1000 10000 -1 $default_filesystem
+5000 10000 -1 btrfs
 	$lvmok{ }
 	method{ format }
 	format{ }
+	label{ KaisenHome }
 	use_filesystem{ }
-	$default_filesystem{ }
+	filesystem{ btrfs }
 	mountpoint{ /home } .
 
--- partman-auto-160.orig/recipes/multi
+++ partman-auto-160/recipes/multi
@@ -5,49 +5,48 @@ partman-auto/text/multi_scheme ::
 	$reusemethod{ }
 	method{ biosgrub } .
 
-512 512 768 ext2
+1024 1024 1024 btrfs
 	$defaultignore{ }
 	method{ format }
 	format{ }
+	label{ KaisenBoot }
 	use_filesystem{ }
-	filesystem{ ext2 }
+	filesystem{ btrfs }
 	mountpoint{ /boot } .
 
-2000 3500 25000 $default_filesystem
+10000 13000 40000 btrfs
 	$lvmok{ }
 	method{ format }
 	format{ }
+	label{ KaisenLinux }
 	use_filesystem{ }
-	$default_filesystem{ }
+	filesystem{ btrfs }
 	mountpoint{ / } .
 
-1000 1500 10000 $default_filesystem
+1000 1500 10000 btrfs
 	$lvmok{ }
 	method{ format }
 	format{ }
+	label{ KaisenVar }
 	use_filesystem{ }
-	$default_filesystem{ }
+	filesystem{ btrfs }
 	mountpoint{ /var } .
 
-100% 512 200% linux-swap
-	$lvmok{ }
-	$reusemethod{ }
-	method{ swap }
-	format{ } .
-
-256 400 2000 $default_filesystem
+256 400 2000 btrfs
 	$lvmok{ }
 	method{ format }
 	format{ }
+	label{ KaisenTmp }
 	use_filesystem{ }
-	$default_filesystem{ }
+	filesystem{ btrfs }
 	mountpoint{ /tmp } .
 
-4000 10000 -1 $default_filesystem
+4000 10000 -1 btrfs
 	$lvmok{ }
 	method{ format }
 	format{ }
+	label{ KaisenHome }
 	use_filesystem{ }
-	$default_filesystem{ }
+	filesystem{ btrfs }
 	mountpoint{ /home } .
 
